components/dashboard/src/projects/Projects.tsx

```diff
@@ -122,27 +122,46 @@ export default function () {
+                                </Link>
+                            </div>
+                            <div className="h-10 px-6 py-1 text-gray-400 text-sm">
+                                <span className="hover:text-gray-600 dark:hover:text-gray-300">
+                                    <Link to={`/`}>
+                                        Branches
+                                    </Link>
+                                </span>
+                                <span className="mx-2 my-auto">·</span>
+                                <span className="hover:text-gray-600 dark:hover:text-gray-300">
+                                    <Link to={`/`}>
+                                        Prebuilds
+                                    </Link>
+                                </span>
                             </div>
-                            <div className="h-1/4 px-6 py-1"><p>__ Active Branches</p></div>
                         </div>
-                        <div className="h-1/6 px-6 border rounded-b-xl dark:border-gray-800 bg-gray-200 cursor-pointer" onClick={() => viewAllPrebuilds(p)}>
+                        <div className="h-10 px-4 border rounded-b-xl dark:border-gray-800 bg-gray-100 border-gray-100 dark:bg-gray-800" onClick={() => viewAllPrebuilds(p)}>
                             {lastPrebuilds.get(p.id)
-                                ? (<div className="flex flex-row space-x-3 h-full text-sm">
-                                    <div className={"my-auto rounded-full w-3 h-3 text-sm align-middle " + (true ? "bg-green-500" : "bg-gray-400")}>
-                                        &nbsp;
-                                    </div>
-                                    <div className="my-auto">{lastPrebuilds.get(p.id)!.branch}</div>
-                                    <div className="my-auto text-gray-400">{moment(lastPrebuilds.get(p.id)!.startedAt, "YYYYMMDD").fromNow()}</div>
-                                    <div className="my-auto text-gray-400 flex-grow text-right">View All ⟶</div>
+                                ? (<div className="flex flex-row h-full text-sm justify-between">
+                                    <Link to={`/`} className="flex my-auto group space-x-2">
+                                        <div className={"my-auto rounded-full w-3 h-3 text-sm align-middle " + (true ? "bg-green-500" : "bg-gray-400")}>
+                                            &nbsp;
+                                        </div>
+                                        <div className="my-auto font-semibold text-gray-500">{lastPrebuilds.get(p.id)!.branch}</div>
+                                        <span className="mx-1 my-auto text-gray-600">·</span>
+                                        <div className="my-auto text-gray-400 flex-grow hover:text-gray-800 dark:hover:text-gray-300">{moment(lastPrebuilds.get(p.id)!.startedAt, "YYYYMMDD").fromNow()}</div>
```

Comment:
**issue**: For better space usage we could abbreviate the timestamp information like the following, however, it should be fine to ship this as is as long as the branch is properly truncated to avoid any text overflow.

| BEFORE | AFTER |
|-|-|
| 19 hours ago | 19h |
| 23 minutes ago | 23m |
| 5 days ago | 5d |